src/test/java/seedu/address/model/ModuleTrackerTest.java

```diff
@@ -51,7 +52,8 @@ public void resetData_withDuplicateModules_throwsDuplicateModuleException() {
         List<Module> newModules = Arrays.asList(CS2106_TUT, editedAlice);
         ModuleTrackerStub newData = new ModuleTrackerStub(newModules);
 
-        assertThrows(DuplicateModuleException.class, () -> moduleTracker.resetData(newData));
```

Comment:
I think this can be fixed as we still have duplicate modules but with different rules